### DISCUSSION TITLE
Cant get the SOC,

### DISCUSSION THREAD

**Discussion by albertusQuelle** [tillsteinbach/VWsFriend#691] [Q&A]
Hi, 
i have a Passat GTE. 
Do i need an active paied license (for VW Connect) for VWsFriend? I cant see my SOC

Thx

**Comment by tillsteinbach** [comment]
VWsFriend can only see the data that you see in the VW App. If your license does not include providing the SoC VWsFriend will also not show it.

**Comment by albertusQuelle** [comment]
Oh that’s a pity. Had many years Hyundai Ioniq and tucson. All nice features „all in“ at Hyundai for free. Does not matter if new or used car.We have a 2 years old used Passat GTE. Now I know why the German automotive industry will die over the time. 149€ per Year! Just to read a SOC. And even less features Had this ok homeassistant and evcc for years integrated with Hyundai. Everyone who reads that, hands away from German brands if you have a fable for Software.For good motor, interior and iron, German cars are still good 😌 Am 16.11.2024 um 13:34 schrieb Till Steinbach ***@***.***>:﻿
VWsFriend can only see the data that you see in the VW App. If your license does not include providing the SoC VWsFriend will also not show it.

**Reply by tillsteinbach** [reply]
Yes, I agree. Best way to make owners unhappy with their cars. It's like a Tax...
I'm lucky to have a new company car every few years.
